BitToGithub/Program.cs

```diff
@@ -50,7 +54,9 @@ static void Main(string[] args)
                     int startIndex = 1;
                     while (true)
                     {
-                        var response = client.GetAsync(bbRepoIssuesUrl + "?start=\{startIndex}").Result;
+                        System.Threading.Thread.Sleep(3000);
```

Comment:
Do we need this?

Reply:
I suppose one sleep is enough... I used the script debugging in break mode to transfer my issues, so I didn't experience the GitHub blocking behavior. Was just erring on the safe side.